packages/devextreme/js/__internal/scheduler/tooltip_strategies/m_mobile_tooltip_strategy.ts

```diff
@@ -65,13 +65,27 @@ export class MobileTooltipStrategy extends TooltipStrategyBase {
     return false;
   }
 
-  _onShowing() {
+  private setTooltipConfig(): void {
     const isTabletWidth = getWidth(getWindow()) > 700;
```

Comment:
Minor comment for refactoring.

Let's move all constants related to sizes to one general place.
It may be: ```__internal/scheduler/device_const.ts``` or something like

Reply:
Yup, agree with that
I left this comment to share this idea + don't forget about it in the future